src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -21,28 +20,24 @@ class NotifyService(
   @Value("\${notify.templates.urgentLicencePrompt}") private val urgentLicencePromptTemplateId: String,
   @Value("\${notify.templates.datesChanged}") private val datesChangedTemplateId: String,
   private val client: NotificationClient,
-  private val pduHeadProperties: PduHeadProperties,
 ) {
   fun sendLicenceApprovedEmail(emailAddress: String, values: Map<String, String>, reference: String) {
     // Hobbled this email - no current requirement to send this out - blanked out the email address to avoid sending
     // sendEmail(licenceApprovedTemplateId, emailAddress, values, reference)
     sendEmail(licenceApprovedTemplateId, "", values, reference)
   }
 
-  fun sendVariationForApprovalEmail(pduCode: String, licenceId: String, firstName: String, lastName: String) {
-    val pduHead = pduHeadProperties.contacts
-      .getOrDefault(pduCode, EmailConfig(forename = "", surname = "", email = "", description = ""))
-
-    if (pduHead.email.isNotBlank()) {
+  fun sendVariationForApprovalEmail(notifyRequest: NotifyRequest, licenceId: String, firstName: String, lastName: String) {
+    if (notifyRequest.email != null && notifyRequest.name != null) {
       val values: Map<String, String> = mapOf(
-        Pair("pduHeadFirstName", pduHead.forename),
+        Pair("pduHeadFirstName", notifyRequest.name),
         Pair("licenceFirstName", firstName),
```

Comment:
As the PDU head name has been combined into one field should we change the template variable from `pduHeadFirstName` to `pduHeadName` and update the templates too?